kythe/go/util/riegeli/golden_test.go

```diff
@@ -89,11 +90,12 @@ func (j *jsonReader) Next() (*spb.Entry, error) {
 func TestGoldenTestData(t *testing.T) {
 	for _, variant := range goldenRiegeliFileVariants {
 		file := strings.Join([]string{goldenRiegeliFilePrefix, variant, "riegeli"}, ".")
-		t.Run(variant, func(t *testing.T) { checkGoldenData(t, file) })
+		opts := strings.Replace(variant, "_", ",", -1)
+		t.Run(variant, func(t *testing.T) { checkGoldenData(t, file, opts) })
 	}
 }
 
-func checkGoldenData(t *testing.T, goldenRiegeliFile string) {
+func checkGoldenData(t *testing.T, goldenRiegeliFile, expectedOptions string) {
```

Comment:
Please document the format of the options string (e.g., is it flags? Space-separated? Other?)

Reply:
It's the magic C++ standard (i.e. CSV of various options).